### PR TITLE
Testing new fields ne-dwmapi-dwmwindowattribute.md

### PR DESCRIPTION
**Pull request by mimisasouvanh**

### CODE REVIEW

**Comment by PRMerger12** [comment]
@mimisasouvanh : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by stevewhims** [comment]
Adding Drew as reviewer, since there was some uncertainty about whether or not we should be documenting these constants (and I can’t remember how it landed). Drew, if this looks ok, do you mind completing the PR?

Many thanks!
Steve

**Comment by drewbatgit** [comment]
@stevewhims Sorry, I should have realized that you would have been auto-assigned to this one. Please just ignore this one as it's just a test. @mimisasouvanh  I'm not sure we need this PR anymore, since we know what the issue is.

File: sdk-api-src/content/dwmapi/ne-dwmapi-dwmwindowattribute.md [hunk 1]
```diff
@@ -141,6 +141,19 @@ Use with [DwmSetWindowAttribute](/windows/win32/api/dwmapi/nf-dwmapi-dwmsetwindo
 
 The maximum recognized <b>DWMWINDOWATTRIBUTE</b> value, used for validation purposes.
 
+### -field DWMWA_USE_IMMERSIVE_DARK_MODE:20
```
Comment (AraHaan) [inline sdk-api-src/content/dwmapi/ne-dwmapi-dwmwindowattribute.md:144]: I thought I already documented this (and the old value of 19) before. Also It is supported before Windows 11 I thought.

Edit:
I did but someone reverted it.

See also: https://github.com/MicrosoftDocs/sdk-api/pull/966

File: sdk-api-src/content/dwmapi/ne-dwmapi-dwmwindowattribute.md [hunk 2]
```diff
@@ -141,6 +141,19 @@ Use with [DwmSetWindowAttribute](/windows/win32/api/dwmapi/nf-dwmapi-dwmsetwindo
...
+
+Use with [DwmSetWindowAttribute](/windows/win32/api/dwmapi/nf-dwmapi-dwmsetwindowattribute). Allows the window frame for this window to be drawn in dark mode colors when the dark mode system setting is enabled. For compatibility reasons, all windows default to light mode regardless of the system setting. The <i>pvAttribute</i> parameter points to a value of type **BOOL**. <b>TRUE</b> to honor dark mode for the window, <b>FALSE</b> to always use light mode.
+
+This value is supported starting with Windows 11 Build 22000.
```
Comment (AraHaan) [inline sdk-api-src/content/dwmapi/ne-dwmapi-dwmwindowattribute.md:148]:
```suggestion
<b>Windows 10 before build 10.0.17763: </b>This value is not supported.

<b>Windows 10 build 10.0.17763 until 10.0.18985: </b>This value is 19 instead (you would have to manually define this in with `#define` or as an constant to pass this in if you want to remember what 19 represents).

<b>Windows 10 build 10.0.18985 or later: </b>This value is supported.
```

**Review by AraHaan — CHANGES_REQUESTED** [review]

File: sdk-api-src/content/dwmapi/ne-dwmapi-dwmwindowattribute.md (hunk 2 above)
Comment (drewbatgit) [reply sdk-api-src/content/dwmapi/ne-dwmapi-dwmwindowattribute.md:148]: @AraHaan  - Please do not update this pull request. This is a test to try to diagnose some issues in our build system. There is an update to these docs coming in the next day or two that has been driven and reviewed by the feature owners. I'm sorry that there was confusion around your updates, but we will not be completing this PR.

Comment (AraHaan) [reply sdk-api-src/content/dwmapi/ne-dwmapi-dwmwindowattribute.md:148]: Thanks for explaining it.

Comment (drewbatgit) [reply sdk-api-src/content/dwmapi/ne-dwmapi-dwmwindowattribute.md:148]: @AraHaan I have a little more info about this. It turns out that the API was added to the header, but was not actually supported in the OS until Windows Build 22000, and that's why your initial documentation was reverted at the time.

Comment (AraHaan) [reply sdk-api-src/content/dwmapi/ne-dwmapi-dwmwindowattribute.md:148]: Also I m guessing that by the part of the docs that says to pass ``TRUE`` would mean that it won't use dark mode if the user configured the system to use light mode. However what if my application code requires to override that option via this method to use dark mode regardless of the system setting? Unless there are plans for that eventually as well (without people like me being forced to custom draw the frame that is).

However I do hate that a lot of Window's built in programs use light mode regardless of the settings (like copying a file via Windows Explorer) as it hurts my eyes seeing the light mode styles in it.

**Pull request closed without merging**